tox.ini

```diff
@@ -108,6 +108,7 @@ commands =
 [molecule_common]
 changedir = {toxinidir}
 deps =
+    ansible-lint==4.2.0
```

Comment:
Does this mean that ansible-lint version 4.2.1 breaks our CI?

Reply:
No, but there is no 4.2.1 in [release history](https://pypi.org/project/ansible-lint/#history).